Team6220_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team6220_CENTERSTAGE/Constants.java

```diff
@@ -0,0 +1,22 @@
+package org.firstinspires.ftc.team6220_CENTERSTAGE;
+
+import org.opencv.core.Scalar;
+import org.opencv.core.Size;
+
+public class Constants {
```

Comment:
Nice job keeping all the constants in one place and adding comments in this file.

Reply:
Thanks!